radar/2024-12-01/spark.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Apache Spark"
```

Comment:
Aha, Apache Spark is back on our radar. :-) We had it on our radar in the past, not sure in which context exactly. In our data space I think indeed it makes good sense to use it. 

Just checking: I would have thought DataBricks would offer it's own data analytics solution? If they do, might be good to explain why we would still want to trial Apache Spark?

Reply:
I think this blip should be in a different new 2025-.. folder and not 2024-12-01?